core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -45,7 +45,7 @@ class AnalyzerManager(
       case Some(moduleId) =>
         f(req, moduleId)
       case None =>
-        sauron ! EnsimeServerError(s"Couldn't find the project for ${fileInfo.file}")
+        sauron forward req
```

Comment:
pretty much :smile: but I think we need a test. Maybe send a request for semantic symbols for `/scala/Int.scala` in the scala-library source jar or something?